src/SmartComponents/BaselinesPage/EditBaseline/helpers.js

```diff
@@ -3,13 +3,13 @@ import jiff from 'jiff';
 import FactKebab from './FactKebab/FactKebab';
 
 /*eslint-disable react/prop-types*/
-function renderKebab({ factName, factValue, fact, isCategory, isSubFact } = {}) {
+function renderKebab({ factName, factValue, factData, isCategory, isSubFact } = {}) {
     return (
         <td>
             <FactKebab
                 factName={ factName }
                 factValue={ factValue }
-                fact={ fact }
+                fact={ factData }
```

Comment:
What is `factData` at this point? We pass in fact name and fact value, which is the fact data

Reply:
It would be factName and factValue unless it is a category, and then it would contain factName and the subFacts.